examples/config.neon

```diff
@@ -2,6 +2,7 @@ extensions:
 	webimages: DotBlue\WebImages\Extension
 
 webimages:
+	umask: 0777 # default FALSE
```

Comment:
Can you please remove this? I don't want this in example configuration. But mention in readme would be nice.